src/drivers/Stresing.py

```diff
@@ -48,7 +48,7 @@ def __init__(self,hardware_params):
         path_dll = folder_path_dll / "stresing" / "ESLSCDLL.dll"
         path_dll = str(path_dll)
 
-        path_config = Path(r"C:\Program Files\Stresing\Escam\config.ini")
+        path_config = Path(r"C:\Program Files\Stresing\Escam\config_UdeM.ini")
```

Comment:
Why did you change the config file name instead of locally changing the config file contents?